src/components/Comments/Comment.js

```diff
@@ -38,6 +46,7 @@ const Comment = ({
   onLongPress,
   updateVote
 }) => {
+  const refSheet = React.useRef();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A `ref` is being created for the BottomSheetMenu component. This is a good practice when you need to control a child component from its parent, such as opening or closing the bottom sheet.
```diff
+ const refSheet = React.useRef();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->